tests/miners/test_llm.py

```diff
@@ -78,6 +78,20 @@ def test_llm_query_handler(self):
         print("--- Interpreted result ---")
         print(interpreted_result)
 
+    def test_edge_cases(self):
```

Comment:
nice, but we need moooore test queries,
for now we know it works only for "Return x tx from my addrexx xyz"
I had broken this code with other prompts related to my address
TODO:
- break it, with strange blockchain prompts